### ISSUE TITLE
Multiple destination point to the same api

### ISSUE THREAD

**Issue by pouki06** [andreytreyt/yarp-swagger#15]
**Description**
I am using a microservice architecture with yarp redirecting all the traffic to the right module. 
Each module is an API and I am using Yarp to unify every module's swagger. In a development process, I'd like to be allowed to debug only the module that I need, and at the same time to beneficiate to the entire API without the need to deploy anything.

When I'm adding another destination in the same cluster with the same prefix, it is not working...

**To Reproduce**
1. Take an API module and run it twice on 2 ports / 2 domains
2. Add this config with 2 destinations : 

```
"ReverseProxy": {
  "Routes": {
    "exemple.api": {
      "ClusterId": "exemple.api",
      "Match": {
        "Path": "/exemple/{**catch-all}"
      },
      "Transforms": [
        {
          "PathRemovePrefix": "/exemple"
        }
      ]
    }
  },
  "Clusters": {
    "exemple.api": {
      "ClusterId": "exemple.api",
      "LoadBalancingPolicy": "FirstAlphabetical",
      "HealthCheck": {
        "Active": {
          "Enabled": "true",
          "Interval": "00:00:05"
        }
      },
      "Metadata": {
        "ConsecutiveFailuresHealthPolicy.Threshold": "1"
      },
      "Destinations": {
        "A-Local": {
          "Address": "http://localhost:5206",
          "Health": "http://localhost:5206/healthz",
          "Swaggers": [
            {
              "PrefixPath": "/exemple",
              "Paths": [
                "/swagger/v1/swagger.json"
              ]
            }
          ]
        },
        "B-Deployed": {
          "Address": "http://192.168.30.100:6533",
          "Health": "http://192.168.30.100:6533/healthz",
          "Swaggers": [
            {
              "PrefixPath": "/exemple",
              "Paths": [
                "/swagger/v1/swagger.json"
              ]
            }
          ]
        }
      }
    }
  }
}
```

3. Launch Yarp
4. See error in "ReverseProxyDocumentFilter.cs" line : `paths.Add($"{swagger.PrefixPath}{key}", value);` because a key already exists

**Expected behavior**
In this case do not try to add another destination because I wish the first available only (the local one if available or the deployed else the local one is not available.

**Other**
I would be happy to provide more information if needed, thanks a lot !

**Comment by andreytreyt** [comment]
@pouki06 Fixed it 🚀 Version [3.2.1](https://www.nuget.org/packages/Treyt.Yarp.ReverseProxy.Swagger/3.2.1) is available. Please star this project ⭐

**Comment by pouki06** [comment]
Wow ! Thank you for the fast fix ! :raised_hands:
